addons/mail/tests/test_res_users_settings.py

```diff
@@ -35,12 +35,10 @@ def test_set_res_users_settings_should_send_notification_on_bus(self):
         with self.assertBus(
                 [(self.cr.dbname, 'res.partner', self.partner_employee.id)],
                 [{
-                    'type': 'mail.record/insert',
+                    'type': 'res.users.settings',
```

Comment:
regression too, what we would want is to introduce the settings model

Reply:
Probably, but not that there's likely plan for other modules to use user settings, and they might probably not be happy to necessarily use discuss models for that.
I'm not quite sure about making a model for `res.users.settings` or not. Decoupling these notifications from `mail.record/insert` was the main goal to ease simplifying the code of Discuss models. We can postpone improvements on `res.users.settings` modelling when we know better what we want and it becomes a higher priority than current PR